### PR TITLE
don't show tokens with 0 balance + refactor

### PR DESCRIPTION
**Pull request by AbstractFruitFactory**
https://radixdlt.atlassian.net/browse/RPNV1-205

### CODE REVIEW

File: src/app/components/wallet/Balance.vue [hunk 1]
```diff
@@ -24,133 +24,142 @@ div
...
-                    return output
-                }, {})
-
-            
-            this.$forceUpdate()
+    update() {
+      const balances = this.identity.account.transferSystem.tokenUnitsBalance
+      this.tokens = Object.entries(balances).reduce(
+        (output, [uri, balance]) => {
+          if (!uri.endsWith('XRD') && balance.equals(0)) {
```
Comment (MuncleUscles) [inline src/app/components/wallet/Balance.vue]: Any number of RRIs can end with `XRD`

File: src/app/components/wallet/Balance.vue [hunk 2]
```diff
@@ -24,133 +24,142 @@ div
...
-    radixTokenManager,
-    RadixTransactionBuilder,
-    RadixAccount,
-    RRI,
-    RadixIdentity,
+  radixTokenManager,
+  RadixTransactionBuilder,
+  RadixAccount,
+  RRI,
+  RadixIdentity
```
Comment (MuncleUscles) [inline src/app/components/wallet/Balance.vue]: I don't know if this is set in tsconfig, but I prefer trailing commas, they make editing and git merges easier https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

**Review by MuncleUscles — CHANGES_REQUESTED** [review]
Overall looks good, problematic uri comparison in one place

File: src/app/components/wallet/Balance.vue (hunk 1 above)
Comment (AbstractFruitFactory) [reply src/app/components/wallet/Balance.vue]: thanks, fixed

File: src/app/components/wallet/Balance.vue (hunk 2 above)
Comment (AbstractFruitFactory) [reply src/app/components/wallet/Balance.vue]: added rule in tslint

**Review by MuncleUscles — APPROVED** [review]

**Pull request merged**